e2e/tests/components/column-picker/column-picker.e2e-spec.ts

```diff
@@ -31,6 +35,55 @@ describe('Column Picker Tests', () => {
         expect(await imageCompare('column-picker-locked')).toEqual(0);
     });
 
+    it('should allow a group to navigate, close and expand using mouse and keyboard', async () => {
+        // close the group
+        const firstGroup = await page.deselectedListGroupBtns.get(0);
+        await firstGroup.click();
+        expect(await imageCompare('column-picker-close-deselected-group-1')).toEqual(0);
+        // expand the group
+        await firstGroup.click();
+        expect(await imageCompare('column-picker-initial')).toEqual(0);
```

Comment:
I would suggest not reusing screenshots. If one fails and not the other, the output could be obscured by the test that runs later. This could make diagnosing failures difficult.

You could probably cut some of the screenshots out of this test without losing coverage. For example, you could assert that the number of elements in the deselected list is consistent with the group being expanded instead of this `imageCompare`.